app/models/concerns/comment/publishing.rb

```diff
@@ -24,7 +24,9 @@ def to_event_stream
         focus_id: focus_id,
         focus_type: focus_type,
         project_id: project_id,
+        category: category,
         section: section,
+        body: body,
```

Comment:
Do we not have to be concerned about the data size here?  Some comments get really large